hangman/game.py

```diff
@@ -1,23 +1,76 @@
 from .exceptions import *
+from random import randint
 
 # Complete with your own, just for fun :)
-LIST_OF_WORDS = []
+LIST_OF_WORDS = ['Python', 'Hello World', 'Women Who Code']
 
 
 def _get_random_word(list_of_words):
     pass
+    if len(list_of_words) != 0:
+       random_index = randint(0, len(list_of_words)-1)
+    else:
+        raise InvalidListOfWordsException()
+    return list_of_words[random_index];
 
 
 def _mask_word(word):
     pass
+    if word != '':
+        letters_of_word = list(word)
+        for letter in letters_of_word:
+            word = word.replace(letter, '*')
```

Comment:
there's also another trick in python where you can multiply strings :)

this means you can simply `return len(word) * "*"` <- this return asterisks of the same length of word